src/main/java/jenkins/plugins/office365connector/model/Fact.java

```diff
@@ -63,4 +64,18 @@ public void setValue(List<String> causesStrList) {
             this.value += cause + ". ";
         }
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
it is used anywhere?

Reply:
I used it in FactsBuilderTest when comparing the expected Fact to the actual Fact that was built.

Reply:
then the question is: do we want to have this code only to satisfy test? IMO it is possible to rewrite test so those methods are not needed

Reply:
I agree, I'll go ahead and change the test so I don't have to implement equals() and hashCode().